src/app/index/index.component.css

```diff
@@ -133,47 +133,34 @@ footer {
   .dropdown-menu-box-homepage {
     margin-top: -15%;
   }
-  }
-
-@media screen and (max-width: 823px) {
-  img#biglogo{
-    margin-left: 61%;
-  }
 }
 
-@media screen and (max-width: 767px) {

```

Comment:
Because of your changes: 
Susper.com:
![image](https://user-images.githubusercontent.com/20185076/29002777-cebf4498-7ac7-11e7-9869-16140b90cb9c.png)
Test link 
![image](https://user-images.githubusercontent.com/20185076/29002775-b6eb4bd2-7ac7-11e7-9564-8a791be4d8c8.png)